packages/runtime-vapor/src/dom/template.ts

```diff
@@ -13,8 +13,18 @@ export function template(html: string) {
 /*! #__NO_SIDE_EFFECTS__ */
 export function children(node: Node, ...paths: number[]): Node {
   for (const idx of paths) {
-    for (let i = 0; i <= idx; i++) {
-      node = node[i === 0 ? 'firstChild' : 'nextSibling']!
+    // In various situations, select the quickest approach.
+    // See https://github.com/vuejs/core-vapor/pull/263
+    switch (idx) {
```

Comment:
In this case, the `switch` statement seems verbose.
Could you also check the performance and bundle size if we use an `if` statement instead?

```suggestion
    if (idx === 0) {
```